### DISCUSSION TITLE
Unity/Deploying-Unity-Game-Server-Into-Playfab-Cloud/

### DISCUSSION THREAD

**Discussion by giscus** [prajwalshettydev/prajwalshetty.github.io#7] [Announcements]
# Unity/Deploying-Unity-Game-Server-Into-Playfab-Cloud/

Deploying a Unity Linux game server on the Microsoft Azure PlayFab service is a great way to host your multiplayer game and manage player data. With Azure PlayFab, game developers can leverage cloud-based infrastructure and services to scale their games and provide an optimal experience for players. In this guide, we will take you through the steps involved in deploying your Unity Linux game server on the PlayFab service, all at no cost.

https://prajwalshetty.com/Unity/Deploying-Unity-Game-Server-Into-Playfab-Cloud/

**Comment by prajwalshettydev** [comment]
Link to playfab official docs: https://learn.microsoft.com/en-us/gaming/playfab/

**Comment by umutercan** [comment]
Hello, I am following the instructions you explained one by one, but each time the playfab server build status is marked as unhealthy. I am using Fishnet. Can you please help me?

**Reply by prajwalshettydev** [reply]
sure, do you have some detailed log?

**Reply by umutercan** [reply]
https://drive.google.com/file/d/1gTR7ky7EW5tDtEioQTz6FfruDrnbHlgQ/view
